src/xml_element.js

```diff
@@ -251,24 +260,25 @@ XMLElement.prototype.addTopActions = function () {
 	this.toggleCollapse.appendChild(document.createTextNode('_'));
 	topActionSpan.appendChild(this.toggleCollapse);
 	
-	var moveDown = document.createElement('span');
-	moveDown.className = 'move_down';
-	moveDown.id = this.domNodeID + '_down';
-	moveDown.appendChild(document.createTextNode('\u2193'));
-	topActionSpan.appendChild(moveDown);
-	
-	var moveUp = document.createElement('span');
-	moveUp.className = 'move_up';
-	moveUp.id = this.domNodeID + '_up';
-	moveUp.appendChild(document.createTextNode('\u2191'));
-	topActionSpan.appendChild(moveUp);
-	
-	var deleteButton = document.createElement('span');
-	deleteButton.className = 'xml_delete';
-	deleteButton.id = this.domNodeID + '_del';
-	deleteButton.appendChild(document.createTextNode('X'));
-	topActionSpan.appendChild(deleteButton);
-	
+	if (this.editor.options.enableEdit) { // Only show menu entries in edit mode, not readonly mode
```

Comment:
There are also keyboard shortcuts that perform these actions which will still be active.  I'd suggest updating the action methods to prevent them from making changes.  See `GUIEditor.prototype.deleteSelected` and similar methods.